search-commons/src/main/java/no/unit/nva/search/IndexingClient.java

```diff
@@ -67,31 +66,27 @@ public IndexingClient(RestHighLevelClientWrapper elasticSearchClient) {
         logger.info(INITIAL_LOG_MESSAGE, ELASTICSEARCH_ENDPOINT_ADDRESS, ELASTICSEARCH_ENDPOINT_INDEX);
     }
 
-    public void addDocumentToIndex(IndexDocument indexDocument) throws SearchException {
-        try {
-            doUpsert(indexDocument.toIndexRequest());
-        } catch (Exception e) {
-            throw new SearchException(e.getMessage(), e);
-        }
+    public Void addDocumentToIndex(IndexDocument indexDocument) throws IOException {
+        elasticSearchClient.index(indexDocument.toIndexRequest(), RequestOptions.DEFAULT);
+        return null;
     }
 
     /**
-     * Removes an document from Elasticsearch index.
+     * Removes a document from Elasticsearch index.
      *
      * @param identifier og document
-     * @throws SearchException when
      */
-    public void removeDocumentFromIndex(String identifier) throws SearchException {
-        try {
-            doDelete(identifier);
-        } catch (Exception e) {
-            throw new SearchException(e.getMessage(), e);
+    public void removeDocumentFromIndex(String identifier) throws IOException {
+        DeleteResponse deleteResponse = elasticSearchClient
```

Comment:
I have inlined the code of doDelete because it is a bit paradoxical a indexing action to throw a Search exception. Better to throw an IOException.